src/webclients/components/sdfv/vscode_sdfv.ts

```diff
@@ -314,6 +319,7 @@ export class VSCodeSDFV extends SDFV {
                     'text': sdfg_array.type + ' properties:',
                 }).appendTo(contents);
 
+                // TODO: Allow container types to be changed here too.
```

Comment:
I wonder if we just shouldn't have users remove and add a new descriptor of a new type. indeed for now TODO(later)